operations.go

```diff
@@ -253,6 +253,21 @@ func (c CmdS3Pull) Resolve() *CommandDefinition {
 }
 func s3PullFactory() Command { return CmdS3Pull{} }
 
+type CmdSetExpansions struct {
+	YamlFile          string `json:"file" yaml:"file"`
```

Comment:
nit: acronyms are usually written in all the same case. For example, [DB instead of Db for database](https://golang.org/pkg/database/sql/#DB).